reference/conanfile/attributes.rst

```diff
@@ -1172,3 +1171,41 @@ be computed. It can take three different values:
    compute the revision for the recipe.
  - ``"scm"``: the commit ID will be used as the recipe revision if it belongs to a known
    repository system (Git or SVN). If there is no repository it will raise an error.
+
+
+.. _python_requires_attribute:
+
+python_requires
+---------------
+
+.. warning::
+
+    This attribute is part of the :ref:`python requires<python_requires>` feature, so
+    it is also an **experimental** feature subject to breaking changes in future releases.
+
+Python requires are associated with the ``ConanFile`` declared in the recipe file, data
+from those imported recipes is accessible using the ``python_requires`` attribute in
+the recipe itself. This attribute is a dictionary where the key is the name of the
+*python requires* reference and the value is a dictionary with the following information:
+
+ - ``ref``: full reference of the python requires
+ - ``exports_folder``: directory in the cache where the exported files are located.
```

Comment:
```suggestion
 - ``exports_folder``: Directory in the cache where the exported files are located.
```

Reply:
After `:` always lower case, except if it is a quote...